Doc/library/poplib.rst

```diff
@@ -31,7 +31,7 @@ mailserver supports IMAP, you would be better off using the
 The :mod:`poplib` module provides two classes:
 
 
-.. class:: POP3(host, port=POP3_PORT[, timeout])
+.. class:: POP3(host, port=POP3_PORT, timeout=socket._GLOBAL_DEFAULT_TIMEOUT)
```

Comment:
I think the current version of this signature looks correct and its documentation states the default value of it correctly:

> (if not specified, the global default timeout setting will be used)

Reply:
Corrected as suggested